doc/release-notes/rl-2605.section.md

```diff
@@ -11,6 +11,8 @@
 
 - `spoof` has been removed, as there are many issues upstream with it working on modern OS versions, and it appears to be unmaintained.
 
+- `nerdctl` has been updated to v2 but there are no obvious breaking changes or a migration guide.
```

Comment:
The release notes have few lines, so this new line often conflicts with other PRs.
I'd like to remove this part in this PR.